packages/swap/src/event-handlers/swapDepositAddressReady.ts

```diff
@@ -43,19 +56,8 @@ export const swapDepositAddressReady = async ({
         srcChain: depositAddress.chain,
       },
     },
-    create: {
-      srcChain: depositAddress.chain,
-      srcAsset: sourceAsset,
-      depositAddress: depositAddress.address,
-      expectedDepositAmount: 0,
-      destAsset: destinationAsset,
-      destAddress: destinationAddress.address,
-      srcChainExpiryBlock: sourceChainExpiryBlock,
-      issuedBlock,
-      channelId,
-      ...rest,
-    },
-    update: {},
+    create: data,
+    update: data,
```

Comment:
Do we need to update the entry if it exists here? I think we are trying to avoid doing this.

Reply:
There was some data missing and since we were not updating at all, the missing data was never filled.
Since what we get from the state chain is the source of truth, I think it makes sense to update the data.
From performance perspective, I don't think it has so much negativity